secant/Resources/Localizable.strings

```diff
@@ -123,7 +123,7 @@
 "sync.message.uptodate" = "Up-To-Date";
 "sync.message.unprepared" = "Unprepared";
 "sync.message.error" = "Error: %@";
-"sync.message.sync" = "%@ Synced";
+"sync.message.sync" = "%@%% Synced";
```

Comment:
I fixed this in #640. We can move the % sign to localisation file. Just please remove it from `SyncStatusSnapshot` so we don't have double percentage sign there.